### PR TITLE
Use native Netlify functions instead of Gatsby functions

### PR DESCRIPTION
**Pull request by cesarvarela**
closes https://github.com/responsible-ai-collaborative/aiid/issues/3017

should be merged after #3005 

Requires the following environment variables to be added to Netlify UI:

```
REALM_API_APP_ID=
REALM_API_GROUP_ID=
REALM_API_PRIVATE_KEY=
REALM_API_PUBLIC_KEY=
REALM_GRAPHQL_API_KEY=
REALM_APP_ID=
API_MONGODB_CONNECTION_STRING=
ROLLBAR_POST_SERVER_ITEM_ACCESS_TOKEN=
```

Can be seen working here: https://pr-158--cesarvarela-staging.netlify.app/apps/submitted/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,10 @@ See [mongo.md](mongo.md)
 
 ## Setting up a development environment
 
+## Important Notice
```
Comment (cesarvarela) [inline README.md:135]: @kepae

File: site/gatsby-site/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+
```
[the remark this answers is not in the document]
Comment (pdcp1) [reply site/gatsby-site/README.md:1]: I'm not sure about the purpose of this README at the `site/gatsby-site` level. Is the intention to replace the main README, or is it for site-specific documentation?

File: site/gatsby-site/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+   GOOGLE_MAPS_API_KEY=
+   GATSBY_AVAILABLE_LANGUAGES=
+   GOOGLE_TRANSLATE_API_KEY=
+   GATSBY_ROLLBAR_TOKEN=
+   CLOUDFLARE_R2_ACCOUNT_ID=
+   CLOUDFLARE_R2_ACCESS_KEY_ID=
+   CLOUDFLARE_R2_SECRET_ACCESS_KEY=
+   CLOUDFLARE_R2_BUCKET_NAME=
+   GATSBY_CLOUDFLARE_R2_PUBLIC_BUCKET_URL=
+   ```
```
Comment (pdcp1) [inline site/gatsby-site/README.md:51]: I know that there may be a lot of information to add, but as a new development team member, I would appreciate more detailed information about each environment variable.
The meaning of each item, whether it is mandatory or not, and where to find the value to input.

File: site/gatsby-site/netlify/functions/lookupIndex.json
Comment (pdcp1) [inline site/gatsby-site/netlify/functions/lookupIndex.json:1]: Is this a fixture file? 🤔

**Comment by cesarvarela** [comment]
> I'm not sure about the purpose of this README at the site/gatsby-site level. Is the intention to replace the main README, or is it for site-specific documentation?

The idea is to have one README per project in the mono repo; it will make it easier to maintain, and it is a common practice.

**Review by pdcp1 — APPROVED** [review]

**Pull request merged**